paddle/fluid/framework/ir/conv_elementwise_add_act_fuse_pass.cc

```diff
@@ -36,10 +36,11 @@ framework::proto::OpDesc PrepareOpDesc(
     const framework::proto::OpDesc& base_desc, const std::string& bias,
     const std::string& activation, const std::string& output) {
   auto proto = base_desc;
+  imperative::StrVarBaseNode empty_str;
```

Comment:
Class `imperative::StrVarBaseNode` is not necessary. We can use `{}` in `desc.SetInput("ResidualData", {})` directly.

Reply:
done